external-crates/move/crates/move-cli/tests/build_tests/simple_new/args.exp

```diff
@@ -3,14 +3,74 @@ External Command `cat P1/Move.toml`:
 [package]
 name = "P1"
 
+# version = "0.1.1"  # semver "maj.min.patch"
+# license = ""       # e.g., "MIT", "GPL", "Apache 2.0"
+# authors = ["..."]  # e.g., ["Joe Smith (joesmith@noemail.com)", "John Snow (johnsnow@noemail.com)"]
```

Comment:
I'd maybe add `edition = "2024.alpha"` or similar here as well since that is becoming a thing :)

Reply:
Done!